langchain/agents/agent.py

```diff
@@ -203,24 +218,31 @@ def _call(self, inputs: Dict[str, str]) -> Dict[str, Any]:
             # If the tool chosen is the finishing tool, then we end and return.
             if isinstance(output, AgentFinish):
                 if self.verbose:
-                    langchain.logger.log_agent_end(output, color="green")
+                    self.callback_manager.on_text(output.log, color="green")
```

Comment:
It would be helpful to me (and I imagine other people?) to know this is the last message of this call to the agent

Reply:
yeah fair point - i think this is a special enough message that we can make it its own method. `on_agent_end` seem good?